# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1459795801 14400
#      Mon Apr 04 14:50:01 2016 -0400
# Node ID 4c61f8313f582b9e5fdaa2d36d81b5bf3933019a
# Parent  590136240f7ccdd1851b44d1fdd178ed351fcd63
* doc/module.mk: Distribute src and "generated" octave.css files.

diff --git a/doc/module.mk b/doc/module.mk
--- a/doc/module.mk
+++ b/doc/module.mk
@@ -324,27 +324,31 @@ DOC_TARGETS += \
   $(srcdir)/doc/interpreter/octave.info \
   doc/interpreter/doc-cache \
   doc/interpreter/octave.ps \
   doc/interpreter/octave.pdf \
   $(OCTAVE_HTML_STAMP) \
   $(HTMLDIR_IMAGES) \
   $(HTMLDIR_CSS)
 
+## Distribute both OCTAVE_CSS and HTMLDIR_CSS so that the rules for
+## building HTMLDIR_CSS work properly.
+
 doc_EXTRA_DIST += \
   $(BUILT_OCTAVE_TEXI_SRC) \
   $(srcdir)/doc/interpreter/octave.info \
   doc/interpreter/TODO \
   doc/interpreter/doc-cache \
   doc/interpreter/octave.dvi \
   doc/interpreter/octave.ps \
   doc/interpreter/octave.pdf \
   doc/interpreter/octave.html \
   $(HTMLDIR_IMAGES) \
-  $(OCTAVE_CSS)
+  $(OCTAVE_CSS) \
+  $(HTMLDIR_CSS)
 
 # Prevent packaging of distribution unless all libraries
 # necessary to create documentation are present
 doc-interpreter-dist-hook:
 	@$(GREP) '#define HAVE_COLAMD 1' $(top_builddir)/config.h > /dev/null || { echo "Documentation creation requires missing COLAMD library.  Cannot package distribution!" ; exit 1; }
 	@$(GREP) '#define HAVE_CHOLMOD 1' $(top_builddir)/config.h > /dev/null || { echo "Documentation creation requires missing CHOLMOD library.  Cannot package distribution!" ; exit 1; }
 	@$(GREP) '#define HAVE_UMFPACK 1' $(top_builddir)/config.h > /dev/null || { echo "Documentation creation requires missing UMFPACK library.  Cannot package distribution!" ; exit 1; }
 	@$(GREP) '#define HAVE_QHULL 1' $(top_builddir)/config.h > /dev/null || { echo "Documentation creation requires missing QHULL library.  Cannot package distribution!" ; exit 1; }
